main.tf

```diff
@@ -153,7 +153,7 @@ locals {
       metadata_backend_url = format(
         "postgres://%s:%s@%s/%s?sslmode=require",
         var.database_username,
-        var.database_password,
+        urlencode(var.database_password),
```

Comment:
QQ: In the outputs.tf ... we print out the non-urlencoded metadata_backend_url.  Does it matter?  I'm guessing not since we don't even need to output that anymore, yes?